src/consensus/pbft_manager.cpp

```diff
@@ -662,7 +662,7 @@ bool PbftManager::stateOperations_() {
   // CHECK IF WE HAVE RECEIVED 2t+1 CERT VOTES FOR A BLOCK IN OUR CURRENT
   // ROUND.  IF WE HAVE THEN WE EXECUTE THE BLOCK
   // ONLY CHECK IF HAVE *NOT* YET EXECUTED THIS ROUND...
-  if (state_ == certify_state && !have_executed_this_round_) {
+  if (!is_syncing_() && state_ == certify_state && !have_executed_this_round_) {
```

Comment:
why not move this "!is_syncing_()" check higher ? For example getVerifiedVotes call sets some internal data that is used in resetRound_() ? Because if not it seems like it is useless call

Reply:
moved